ros-tutorials/lesson-1-basics.md

```diff
@@ -1,23 +1,35 @@
 # Lesson 1 - Basics
 
 ## Summary
-This lesson will introduce you into the very basics of ROS. By the end, you will know how to start ROS and run the turtle simulation. Before starting, you should take a few minutes and just read/watch videos about ROS in general, so you have a general understanding on what it is used for.
+This lesson will introduce you into the very basics of ROS. By the end, you will know how to start ROS and run the turtle simulation. Before starting, you should take a few minutes and just read/watch videos about ROS in general, just so that you have a general understanding of its uses.
 
 ## Vocab
-- ROS: Robot Operating System, the software we are learning about
-- `roscore`: Command used to start ROS
-- turtlesim: A ROS package that displays a turtle and allows basic interactions
+- **ROS**: Robot Operating System, the software we are learning about
+- **`roscore`**: Command used to start ROS
+- **turtlesim**: A ROS package that displays a turtle and allows basic interactions
 
 ## Goals
-1. Start the roscore
-2. Display the turtle simulation
+1. Start the roscore.
+2. Print information about a node.
+3. Display the turtle simulation.
 
 ## Resources
 - [Understanding ROS Nodes](http://wiki.ros.org/ROS/Tutorials/UnderstandingNodes)
-- [Turtlesim](http://wiki.ros.org/turtlesim)
 
 ## Notes
-- Before running ros commands, you need to run `source /opt/ros/kinetic/setup.bash` inside the container. This will set up your environment to run ROS commands, and has to be run in every new terminal
-- The version of ROS you are using is kinetic
-- If you are done with roscore, hit `ctrl-c` to exit the command
-- Whenever you want to run ROS commands, you need to be in the container. This means that if you create a new terminal, you have to ssh into the docker container you are using
+- :information_source: Before running ROS commands, you need to run
+    ```bash
+    source /opt/ros/melodic/setup.bash
+    ```
+    inside the container. This will set up your environment to run ROS commands, and has to be run in every new terminal.
+- :information_source: Run this command inside the container to install the tutorial packages:
+    ```bash
+    sudo apt-get update && sudo apt-get install ros-melodic-ros-tutorials
```

Comment:
I don't think you need sudo anymore with root access. Also I prefer commands not to be chained with ands, but it's not a big deal if it is.

Reply:
Good points on both, will incorporate. Also will change to apt, as that is recommended now.